src/lib/components/about/officer-profile.svelte

```diff
@@ -4,21 +4,17 @@
   import { termIndex } from '$lib/stores/term-index';
 
   export let info: Officer;
-  export let placeholderPicture = 'placeholder.png';
+  export let placeholderPicture = 'placeholder.webp';
   export let dev = false;
 
   const officerName = info.fullName ?? '';
   const officerPicture = info.picture ?? placeholderPicture;
 
   $: titleHTML = info.positions[VISIBLE_TERMS[$termIndex]].title
-    .replace(/Create\s/, `<span class="brand-em brand-pink">Create </span>`)
-    .replace(/Algo\s/, `<span class="brand-em brand-purple">Algo </span>`)
-    .replace(/Dev\s/, `<span class="brand-em brand-bluer">Dev </span>`)
-    .replace(
-      /NodeBuds/,
-      `<span class="headers">node<span class="brand-em brand-red">Buds</span></span>`
-    );
-
+    .replace(/Algo\s/, `<span class="brand-bold acm-purple">Algo&nbsp</span>`)
+    .replace(/Create\s/, `<span class="brand-bold acm-pink">Create&nbsp</span>`)
+    .replace(/Dev\s/, `<span class="brand-bold acm-bluer">Dev&nbsp</span>`)
+    .replace(/nodebuds\s/, `<span class="brand-header">Node Buds&nbsp</span>`);
```

Comment:
Nit: End your HTML entities with a semicolon.
```suggestion
    .replace(/Algo\s/, `<span class="brand-bold acm-purple">Algo&nbsp;</span>`)
    .replace(/Create\s/, `<span class="brand-bold acm-pink">Create&nbsp;</span>`)
    .replace(/Dev\s/, `<span class="brand-bold acm-bluer">Dev&nbsp;</span>`)
    .replace(/nodebuds\s/, `<span class="brand-header">Node Buds&nbsp;</span>`);
```

Reply:
Fixed in commit 8a4f48a